jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/GeneratorManagerTest.java

```diff
@@ -20,17 +20,18 @@
 import org.eclipse.jkube.kit.common.KitLogger;
 import org.eclipse.jkube.kit.config.image.ImageConfiguration;
 import org.eclipse.jkube.kit.config.resource.ProcessorConfig;
-
-import mockit.Mocked;
-import mockit.Verifications;
 import org.junit.Before;
 import org.junit.Test;
-
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
 import static org.assertj.core.api.Assertions.assertThat;
-
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Don't use this `@RunWith` annotation, it would require additional dependency in JUnit5 migration as well. Instead instantiate these mock objects in a `@Before` `setUp` method